cross/bcrypt/Makefile

```diff
@@ -11,4 +11,6 @@ HOMEPAGE = https://github.com/pyca/bcrypt
 COMMENT  = Modern password hashing for your software and your servers
 LICENSE  = Apache 2.0
 
+PLIST_TRANSFORM = sed -e 's%@PYTHON_SITE_PACKAGES@%$(PYTHON_LIB_DIR)/site-packages%' -e 's%@PYTHON_VERSION@%$(SPK_SHORT_VERS)%'
```

Comment:
Thinking of this again I wonder if it shouldn't just be part of `cross/python3`?

Reply:
Actually, if anything, it should be part of some .mk import files I believe.

Reply:
Just checked and it cannot "just" be put in the `cross/python3` logic. At this level it "could" have been inserted in the auto-generated `python-cc.mk` file but this file is imported **after** the one which defines the default `PLIST_TRANSFORM` value (which is `mk/spksrc.plist.mk` from what I see).

If we are to go towards the `cross/python3` approach I guess the ordering of  https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.python-module.mk#L17-L20 and https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.python-wheel.mk#L17-L20 lines need to be inverted.